calico_node/filesystem/etc/rc.local

```diff
@@ -27,8 +27,10 @@ mkdir /etc/service/enabled
 # dirs contained in Docker image. This is due to bug in Docker with graphdriver
 # overlay on CentOS 7.X kernels (https://github.com/docker/docker/issues/15314)
 
-# Felix is always enabled
-cp -a /etc/service/available/felix /etc/service/enabled/
+# In certain unique situation we want to disable felix
```

Comment:
How about adding an example:  `# Allow Felix to be disabled, for example, if the user is running Felix outside the container.`